shell/src/main/java/org/apache/accumulo/shell/commands/ConfigCommand.java

```diff
@@ -129,8 +131,7 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
       // check for deprecation
       var theProp = Property.getPropertyByKey(property);
       if (theProp != null && theProp.isDeprecated()) {
-        if (!forceSet(shellState, cl,
-            "Trying to set deprecated property `" + property + "` continue")) {
+        if (!shellState.yorn("Trying to set deprecated property `" + property + "` continue")) {
```

Comment:
The old method handled the force option. This change only does the prompt. You've got to do something like: `!isForceSet() && !shellstate.yorn(...)`. You could keep the existing `forceSet()` method, but change it's behavior to let the prompt happen on `shellState` instead of inside that method.